### DISCUSSION TITLE
manage mutiple arduino device in one server

### DISCUSSION THREAD

**Discussion by prostoneM** [JAndrassy/ArduinoOTA#166] [Q&A]
Hi juraj, I apprecite to make this library.
This library is very usefull to use arduino.
I have a question to use multiple devices. 
First, I use arduino nano 33 iot
I want to upload the code labeled for each device to each device at once.
Is it possible?

**Comment by JAndrassy** [comment]
just write a script on the computer using the upload tool by Arduino. the problem has nothing to do with my library
